java/src/jmri/jmrix/loconet/SlotManager.java

```diff
@@ -1210,6 +1245,19 @@ public void readCVOpsMode(String CVname, jmri.ProgListener p, int addr, boolean
         doRead(CV, p, 0x2F);  // although LPE implies 0x2C, 0x2F is observed
     }
 
+    /**
+     * Read a CV via the Service-mode programmer.
+     *
+     * @param CV the CV number
+     * @param p programmer
+     * @throws jmri.ProgrammerException if an unsupported programming mode is exercised
+     */
+    @Deprecated // 4.1.1
+    public void readCV(int CV, jmri.ProgListener p) throws jmri.ProgrammerException {
```

Comment:
This seems to have lost the changes in PR #6376.  That removed this method and merged it into the call just above.